src/consensus/aft/raft.h

```diff
@@ -873,7 +893,7 @@ namespace aft
     bool can_replicate_unsafe()
     {
       return state->leadership_state == kv::LeadershipState::Leader &&
-        !retirement_committable_idx.has_value();
+        !is_retired_committed();
```

Comment:
Used to decide if signatures should be emitted, and in the frontend when accepting user transactions.